openhtf/util/test.py

```diff
@@ -197,9 +197,6 @@ def _handle_test(self, test):
     test.AddOutputCallbacks(
         lambda record: setattr(record_saver, 'record', record))
 
-    # Disable the station_api for unit tests.
```

Comment:
hm, i need to remember to add in a `conf.Load(station_api_port=None)` to have the same effect.